### PR TITLE
Low contrast label fix

### LINKED ISSUE #1591: Too Low Contrast in History of Transactions View When No Transactions
I can barely see the text "No transactions found".
See 
poor contrast

### PR DESCRIPTION
**Pull request by Jujhar**
Darkened the grey to a close black for the _No Transactions Found_ label.

image

Fixes #1591

### CODE REVIEW

**Comment by alexlyp** [comment]
Please use 091440 instead of the one you chose.

**Comment by Jujhar** [comment]
Updated, should be ready to go now.

Previously it did not overlap if window was at a certain width now it will not overlap at all.

**Comment by matheusd** [comment]
Please look at the latest specs as pointed out in #1582 

https://xd.adobe.com/spec/5a5726c5-e6ea-4273-6262-82582819feec-2c66/screen/f18f4868-bc65-4642-ae12-a7611037d606/Tickets-My-Tickets-Empty-state-3/

**Comment by Jujhar** [comment]
Updated. I changed the colour as well as according to the specs.

File: app/components/indicators/NoTransactions.js [hunk 1]
```diff
@@ -3,5 +3,8 @@ import { FormattedMessage as T } from "react-intl";
 export default () => (
   <div className="no-transactions-indicator">
     <T id="noTransactions.description" m="No Transactions Found" />
+    <br />
```
Comment (matheusd) [inline app/components/indicators/NoTransactions.js]: Please don't use `<br>` for layout, use a proper css style such as `margin`.

**Comment by matheusd** [comment]
Also notice that in the specs require that the title and filters not be shown when there are no transactions.

File: app/components/views/TransactionsPage/HistoryTab/Page.js [hunk 2]
```diff
@@ -40,7 +40,9 @@ const Page = ({
       <div className="history-content-title-text">
         <T id="history.title" m="Transaction History" />
       </div>
-      <div className="history-select-tx-types-area">
+      <div className="history-select-tx-types-area" style={transactions.length == 0
+        ? { display: "none" }
+        : { display: "initial" } } >
```
Comment (alexlyp) [inline app/components/views/TransactionsPage/HistoryTab/Page.js]: We don't use inline style

**Comment by Jujhar** [comment]
Updated those please have another look.

**Comment by alexlyp** [comment]
You're still using an inline style.

File: app/components/views/TransactionsPage/HistoryTab/Page.js (hunk 2 above)
Comment (alexlyp) [reply app/components/views/TransactionsPage/HistoryTab/Page.js]: Well I suppose we do use inline styles in certain applications.  But I'd rather this be 

`style{{display: transactions.length == 0 ? "none" : "initial"}}`

Comment (Jujhar) [reply app/components/views/TransactionsPage/HistoryTab/Page.js]: OK great think its ready now.

**Comment by Jujhar** [comment]
I ran lint-fix and it passed

File: app/style/MiscComponents.less [hunk 3]
```diff
@@ -647,7 +652,15 @@
 }
 
 .no-transactions-indicator {
+  color: #617386;
```
Comment (alexlyp) [inline app/style/MiscComponents.less]: Where did this color come from?  You need to use the design specs to choose the proper color.

Comment (Jujhar) [reply app/style/MiscComponents.less]: That is from the specs, it is the color of the text.
https://xd.adobe.com/spec/5a5726c5-e6ea-4273-6262-82582819feec-2c66/screen/f18f4868-bc65-4642-ae12-a7611037d606/Tickets-My-Tickets-Empty-state-3/

Comment (alexlyp) [reply app/style/MiscComponents.less]: Gotcha, just didn't recognize it.

**Pull request merged**